oap-server/server-starter/src/main/resources/ui-initialized-templates/kong/kong-service.json

```diff
@@ -54,7 +54,7 @@
                     "showTableValues": true,
                     "tableHeaderCol1": "Instances",
                     "tableHeaderCol2": "Reachable",
-                    "decorations": {
+                    "valueMappings": {
                       "0": "unreachable",
                       "1": "reachable"
```

Comment:
Are these simple 0 works? Or we need to update to regex?

Reply:
https://github.com/apache/skywalking-booster-ui/pull/422#issue-2596458719 It works, `0/1` is enough.